lib/tasks/seed.rake

```diff
@@ -31,6 +31,6 @@ namespace :db do
   end
 
   # Override db:seed to run all the common and environments seeds plus the original db:seed.
-  desc "Load the seed data from db/seeds.rb, db/seeds/#{Seedbank.matcher} and db/seeds/ENVIRONMENT/#{Seedbank.matcher}. ENVIRONMENT is the current environment in Rails.env."
+  desc "Load the seed data from db/seeds.rb, db/seeds/#{Seedbank.matcher} and db/seeds/ENVIRONMENT/#{Seedbank.matcher} (ENVIRONMENT is the current RAILS_ENV)."
```

Comment:
I renamed it a little bit because I noticed other `db` tasks, e.g. `db:create`, use `RAILS_ENV` in comment.
So this way it will look more native I think.
Dots at the end are also would be striped by rake but I didn't remove them.